gtfs-realtime/spec/en/trip-updates.md

```diff
@@ -4,6 +4,10 @@ Trip updates represent fluctuations in the timetable. We would expect to receive
 
 There should be **at most** one trip update for each scheduled trip. In case there is no trip update for a scheduled trip, it will be concluded that no realtime data is available for the trip. The data consumer should **not** assume that the trip is running on time.
 
+If a vehicle is serving multiple trips within the same block (for more information about trips and blocks, please refer to [GTFS trips.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#tripstxt)):
+* the feed should include a TripUpdate for the trip currently being served by the vehicle as well as a another TripUpdate for the next trip in the block. Including multiple TripUpdates for the same vehicle avoids prediction "pop-in" for riders as the vehicle transitions from one trip to another and also gives riders advance notice of delays that impact downstream trips (e.g., when the known delay exceeds planned layover times between trips).
+* the respective TripUpdate entities are not required to be added to the feed in the same order that they are scheduled in the block. For example, if there are trips with `trip_ids` 1, 2, and 3 that all belong to one block, and the vehicle travels trip 1, then trip 2, and then trip 3, the `trip_update` entities may appear in any order - for example, adding trip 2, then trip 1, and then trip 3 is allowed.
```

Comment:
Is there a reason why not encourage this production? I think if we would suggest tripUpdates should be sorted in departure order we will cover all situation, won't we?

Reply:
@skinkie Good question - although this particular PR isn't intended to address block ordering of TripUpdates (the 2nd bullet point shows up in the diff because I re-organized the text). There was previous discussion on the block ordering of TripUpdates in https://github.com/google/transit/pull/176, and some pushback on encouraging they be sorted. If you'd like to discuss further I suggest we open another issue focused just on that.

Reply:
+1 lets do that.

Reply:
Issue opened for discussion at https://github.com/google/transit/issues/207.